### PR TITLE
show time or date while create or update element

### PR DESCRIPTION
**Pull request by Walid-Sahnoun**
Signed-off-by: walid-sahnoun <walid.sahnoun@rte-france.com>

### CODE REVIEW

File: src/components/directory-content.js [hunk 1]
```diff
@@ -120,6 +120,7 @@ const DirectoryContent = () => {
 
     const classes = useStyles();
     const intl = useIntl();
+    const currentDay = useRef(new Date().setHours(0, 0, 0, 0));
```
Comment (laurgarn) [inline src/components/directory-content.js]: I would prefer `toDayMidnight`

File: src/components/directory-content.js [hunk 2]
```diff
@@ -344,7 +345,14 @@ const DirectoryContent = () => {
     function dateCellRender(cellData) {
         const data = new Date(cellData.rowData[cellData.dataKey]);
         if (data instanceof Date && !isNaN(data)) {
-            const date = data.toLocaleDateString(intl.locale);
+            const copiedDate = new Date(data).setHours(0, 0, 0, 0);
```
Comment (laurgarn) [inline src/components/directory-content.js]: I would prefer `cellMidnight`

File: src/components/directory-content.js [hunk 3]
```diff
@@ -344,7 +345,14 @@ const DirectoryContent = () => {
...
+            const time = new Intl.DateTimeFormat(intl.locale, {
+                timeStyle: 'medium',
+            }).format(data);
+            const date =
```
Comment (laurgarn) [inline src/components/directory-content.js]: I would prefer something like `cellText`

File: src/components/directory-content.js (hunk 1 above)
Comment (Walid-Sahnoun) [reply src/components/directory-content.js]: done

File: src/components/directory-content.js (hunk 2 above)
Comment (Walid-Sahnoun) [reply src/components/directory-content.js]: ok

File: src/components/directory-content.js (hunk 3 above)
Comment (Walid-Sahnoun) [reply src/components/directory-content.js]: ok

**Comment by ayolab** [comment]
Tests OK

**Review by ayolab — APPROVED** [review]

**Review by laurgarn — APPROVED** [review]

**Pull request merged**